EventFilter/EcalRawToDigi/plugins/EcalRawToDigiGPU.cc

```diff
@@ -97,6 +85,40 @@ void EcalRawToDigiGPU::acquire(edm::Event const& event,
   edm::Handle<FEDRawDataCollection> rawDataHandle;
   event.getByToken(rawDataToken_, rawDataHandle);
 
+  // scratch
+  ecal::raw::ScratchDataGPU scratchGPU = {
+    cms::cuda::make_device_unique<uint32_t[]>(2, ctx.stream())
+  };
+
+  // input cpu data
+  ecal::raw::InputDataCPU inputCPU = {
+    cms::cuda::make_host_unique<unsigned char[]>(
```

Comment:
comment for the future (after CMSSW 11.1.0 / CUDA 11.0 / c++17): would it make sense to use `std::byte` instead of `unsigned char` ?

Reply:
I would say we should be consistent with what sits inside of the FEDRawData... although we do make a copy before sending to the device...